src/modules/commander/worker_thread.cpp

```diff
@@ -174,14 +174,26 @@ void WorkerThread::threadEntry()
 		_ret_value = 0;
 		break;
 
-	case Request::ParamResetSensorFactory:
-		const char *reset_cal[] = { "CAL_ACC*", "CAL_GYRO*", "CAL_MAG*" };
-		param_reset_specific(reset_cal, sizeof(reset_cal) / sizeof(reset_cal[0]));
-		_ret_value = param_save_default();
+	case Request::ParamResetSensorFactory: {
+			const char *reset_cal[] = { "CAL_ACC*", "CAL_GYRO*", "CAL_MAG*" };
+			param_reset_specific(reset_cal, sizeof(reset_cal) / sizeof(reset_cal[0]));
+			_ret_value = param_save_default();
 #if defined(CONFIG_BOARDCTL_RESET)
-		px4_reboot_request(false, 400_ms);
+			px4_reboot_request(false, 400_ms);
 #endif // CONFIG_BOARDCTL_RESET
-		break;
+			break;
+		}
+
+	case Request::ParamResetAllConfig: {
+			const char *exclude_list[] = {
+				"LND_FLIGHT_T_HI",
+				"LND_FLIGHT_T_LO",
+				"COM_FLIGHT_UUID"
+			};
+			param_reset_excludes(exclude_list, 3);
```

Comment:
```suggestion
			param_reset_excludes(exclude_list, sizeof(exclude_list) / sizeof(exclude_list[0]));
```

Reply:
Can you change this? It's a potential source of inconsistency when updating.

Reply:
Done!